### ISSUE TITLE
Issue Deleting

### ISSUE THREAD

**Issue by peetersdaan** [luluhoc/home-cinema-toolkit#108]
First of all thanks for creating the tool!

I recently installed it using Docker on a Synology NAS. I am able to search for the movies and it has deleted quite a few movies. 

Currently I have a task that deletes all the movies <7 at 00:00. However, when I go to my Plex library, or search in HCT, I still find quite some movies 7<. 

When I search for the movies in HCT and try to delete selected movies I get the error message: Issue Deleting

![](https://user-images.githubusercontent.com/54709869/96184048-e5d31080-0f05-11eb-9a16-5a56025bc055.png)

Log from HCT:
`      
    }
  },
  response: {
    status: 404,
    statusText: 'Not Found',
    headers: {
      connection: 'close',
      date: 'Sun, 04 Apr 2021 07:06:08 GMT',
      'content-type': 'application/json; charset=utf-8',
      server: 'Kestrel',
      'cache-control': 'no-cache, no-store, must-revalidate, max-age=0',
      pragma: 'no-cache',
      'transfer-encoding': 'chunked',
      expires: '0'
    },
    config: {
      url: 'http://172.17.0.1:32783/api/v3/movie/1551?addImportExclusion=true&deleteFiles=true',
      method: 'delete',
      headers: [Object],
      transformRequest: [Array],
      transformResponse: [Array],
      timeout: 0,
      adapter: [Function: httpAdapter],
      xsrfCookieName: 'XSRF-TOKEN',
      xsrfHeaderName: 'X-XSRF-TOKEN',
      maxContentLength: -1,
      maxBodyLength: -1,
      validateStatus: [Function: validateStatus],
      data: undefined
    },
    request: <ref *1> ClientRequest {
      _events: [Object: null prototype],
      _eventsCount: 7,
      _maxListeners: undefined,
      outputData: [],
      outputSize: 0,
      writable: true,
      destroyed: false,
      _last: true,
      chunkedEncoding: false,
      shouldKeepAlive: false,
      useChunkedEncodingByDefault: false,
      sendDate: false,
      _removedConnection: false,
      _removedContLen: false,
      _removedTE: false,
      _contentLength: 0,
      _hasBody: true,
      _trailer: '',
      finished: true,
      _headerSent: true,
      socket: [Socket],
      _header: 'DELETE /api/v3/movie/1551?addImportExclusion=true&deleteFiles=true HTTP/1.1\r\n' +
        'Accept: application/json, text/plain, */*\r\n' +
        'User-Agent: request\r\n' +
        'X-Api-Key: 414e95b76a1c4cb28f8f5b7045a6a8af\r\n' +
        'Host: 172.17.0.1:32783\r\n' +
        'Connection: close\r\n' +
        '\r\n',
      _keepAliveTimeout: 0,
      _onPendingData: [Function: noopPendingOutput],
      agent: [Agent],
      socketPath: undefined,
      method: 'DELETE',
      maxHeaderSize: undefined,
      insecureHTTPParser: undefined,
      path: '/api/v3/movie/1551?addImportExclusion=true&deleteFiles=true',
      _ended: true,
      res: [IncomingMessage],
      aborted: false,
      timeoutCb: null,
      upgradeOrConnect: false,
      parser: null,
      maxHeadersCount: null,
      reusedSocket: false,
      host: '172.17.0.1',
      protocol: 'http:',
      _redirectable: [Writable],
      [Symbol(kCapture)]: false,
      [Symbol(kNeedDrain)]: false,
      [Symbol(corked)]: 0,
      [Symbol(kOutHeaders)]: [Object: null prototype]
    },
    data: {
      message: 'Movie with ID 1551 does not exist',
      description: 'NzbDrone.Core.Datastore.ModelNotFoundException: Movie with ID 1551 does not exist\n' +
        '   at NzbDrone.Core.Datastore.BasicRepository`1.Get(Int32 id) in D:\\a\\1\\s\\src\\NzbDrone.Core\\Datastore\\BasicRepository.cs:line 96\n' +
        '   at NzbDrone.Core.Movies.MovieService.DeleteMovie(Int32 movieId, Boolean deleteFiles, Boolean addExclusion) in D:\\a\\1\\s\\src\\NzbDrone.Core\\Movies\\MovieService.cs:line 214\n' +
        '   at Radarr.Http.REST.RestModule`1.<set_DeleteResource>b__28_0(Object options) in D:\\a\\1\\s\\src\\Radarr.Http\\REST\\RestModule.cs:line 103\n' +
        '   at Nancy.NancyModule.<>c__DisplayClass8_0`1.<Delete>b__0(Object args)\n' +
        '   at Nancy.NancyModule.<>c__DisplayClass10_0`1.<Delete>b__0(Object args, CancellationToken ct)\n' +
        '   at Nancy.Routing.Route`1.Invoke(DynamicDictionary parameters, CancellationToken cancellationToken)\n' +
        '   at Nancy.Routing.DefaultRouteInvoker.Invoke(Route route, CancellationToken cancellationToken, DynamicDictionary parameters, NancyContext context)\n' +
        '   at Nancy.Routing.DefaultRequestDispatcher.Dispatch(NancyContext context, CancellationToken cancellationToken)\n' +
        '   at Nancy.NancyEngine.InvokeRequestLifeCycle(NancyContext context, CancellationToken cancellationToken, IPipelines pipelines)'
    }
  },
  isAxiosError: true,
  toJSON: [Function: toJSON]
}
`

**Comment by luluhoc** [comment]
It is simply the bug with tasks right?

**Comment by peetersdaan** [comment]
Not exactly. I get this error message when manually deleting the movies on the search ratings/page.

**Comment by triksmelb** [comment]
Seeing this error once unwanted movies are selected and I press delete.
Movies (approx 200 results returned with 6.5 rating setting) are manually selected and I click the delete icon.
Error appears at bottom right in red after 5 or so seconds.
Can post logs but not sure which part and don't want to post the entire  40,000 lines.

Synology DSM 6.2.3-25426 Update 3
Synology Docker 18.09.0-0519
HCT (Docker / lulu45-home-cinema-toolkit) v1.2.1
